packages/@storefront/breadcrumbs/test/unit/refinement-crumbs.ts

```diff
@@ -27,28 +28,78 @@ suite('RefinementCrumbs', ({ expect, spy, stub, itShouldProvideAlias }) => {
       expect(updateState).to.be.called;
     });
 
-    it('should set the navigationSelector and selectedRefinementsUpdated based on SEARCH storeSection', () => {
-      const select = (refinementCrumbs.select = spy());
-      refinementCrumbs.props.storeSection = StoreSections.SEARCH;
-      refinementCrumbs.updateState = () => null;
-      refinementCrumbs.init();
+    describe('SEARCH storeSection', () => {
+      beforeEach(() => {
+        refinementCrumbs.props.storeSection = StoreSections.SEARCH;
+        refinementCrumbs.updateState = () => null;
+      });
+
+      it('should set the navigationSelector and selectedRefinementsUpdated', () => {
+        const select = (refinementCrumbs.select = spy());
```

Comment:
Will the linter complain if we remove these parentheses? I think these parentheses were a product of having prettier.

Reply:
Commented above. Resolving conversation.